.ruby-version

```diff
@@ -1 +1 @@
-2.4.5
+ruby-3.0.0
```

Comment:
This is one of the things I wanted to change, probably this file shouldn't be here and the gem should require a minimum 2.x version

Reply:
Ok, I've removed the `.ruby-version`, and the gem is requiring a ruby version >= 2.4: https://github.com/mirubiri/address_composer/blob/f09c96ccfb2c4f94beded48b6ffb87cd5ce59bee/address_composer.gemspec#L26